navigation/ProyectoSegmentacion/scripts/AuxCode.py

```diff
@@ -0,0 +1,69 @@
+
+#! /usr/bin/env python
+
+import rospy
+from nav_msgs.msg import Odometry
+# Brings in the SimpleActionClient
+import actionlib
+# Brings in the .action file and messages used by the move base action
+from move_base_msgs.msg import MoveBaseAction, MoveBaseGoal
+from actionlib_msgs.msg import *
+from geometry_msgs.msg import Pose, Point, Quaternion, Twist,Vector3
+from move_base_msgs.msg import MoveBaseAction, MoveBaseGoal
+from tf2_geometry_msgs.msg import *
+from tf.transformations import quaternion_from_euler
+from visualization_msgs.msg import Marker
+from math import radians, pi
+def movebase_client():
```

Comment:
Dejá dos espacios luego de los imports.

```suggestion


def movebase_client():
```